upload_form.py

```diff
@@ -459,8 +550,11 @@ def set_columns(filename, fileformat=None):
                                                     branch=branch_database,
                                                    )
     except Exception as ex:
-        return render_template('error.html', error=str(ex),
-                               traceback=traceback.format_exc(ex))
+        if testmode:
+            raise ex
+        else:
+            return render_template('error.html', error=str(ex),
```

Comment:
I'm not sure I understand what you're asking - is this a problem with the `render_template` command here?  If so, could you state the full command you think should be in place?  I'm not sure where `, None` belongs